Demos/Blazorise.Demo/Layouts/MainLayout.razor

```diff
@@ -165,35 +170,8 @@
                     </BarStart>
                     <BarEnd>
                         <BarItem>
-                            <BarDropdown>
```

Comment:
Not sure about removing this from the menu. I use it to quickly test theme generator when I change something. I think it's best to return this and then add additional button to open theme builder.

Reply:
Couldn't that be made in the new page too? I thought it was redundant then. But if you insist, I'll return it.

Reply:
Having it on a new page means I would need to navigate for every change. This is useful for quick theme setup

Reply:
That's what the instant preview card is for, but it is not ready. Anyway, I'll undo the deletion of that item

Reply:
Done